x-pack/plugins/search_notebooks/public/components/notebooks_button.tsx

```diff
@@ -47,7 +47,7 @@ export const SearchNotebooksButton = ({
   }
   return (
     <EuiButtonEmpty
-      color="success"
+      color="accentSecondary"
```

Comment:
Same comment as above

Reply:
accentSecondary:
![CleanShot 2024-12-13 at 09 13 05@2x](https://github.com/user-attachments/assets/2991f9ee-6680-47dd-a557-452c2b18f27a)

primary:
![CleanShot 2024-12-13 at 09 13 59@2x](https://github.com/user-attachments/assets/92c561ea-7131-4e80-ad62-f27e95deff54)

text:
![CleanShot 2024-12-13 at 09 14 37@2x](https://github.com/user-attachments/assets/0fea91fa-c2f3-4670-a5fc-15c7131ef110)

cc/ @julianrosado @daveyholler for awareness